docker-storage-setup.conf

```diff
@@ -1,20 +1,21 @@
-# Specify storage driver one wants to use with docker. Default is devicemapper.
+# Specify storage driver one wants to use with container runtimes.
+# Default is devicemapper.
 # Other possible options are overlay, overlay2 and "". Empty string means do
 # not do any storage setup.
 STORAGE_DRIVER=devicemapper
 
-# A set of extra options that should be appended to the generated
-# DOCKER_STORAGE_OPTIONS string. These options will be passed to the Docker
-# daemon as-is and should be valid Docker storage options.
-# EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.fs=ext4"
+# Set extra options that will be appended to the generated STORAGE_OPTIONS
```

Comment:
ok, can we make this change when you actually introduce option EXTRA_STORAGE_OPTIONS. Right now we are modifying man page without actually code supporting what man page says.